doc/R-package/mnistCompetition.md

```diff
@@ -259,11 +258,11 @@ model <- mx.model.FeedForward.create(lenet, X=train.array, y=train.y,
 
 ```
```

Comment:
t hi s is auto generated by knitt
On Mon, Oct 19, 2015 at 10:50 PM Mu Li <notifications@github.com> wrote:

> In doc/R-package/mnistCompetition.md
> <https://github.com/dmlc/mxnet/pull/333#discussion_r42458228>:
>
> > @@ -259,11 +258,11 @@ model <- mx.model.FeedForward.create(lenet, X=train.array, y=train.y,
> >
> >  ```
>
> also use ```r for results? its font size is much larger than the r codes
>
>
> http://mxnet.readthedocs.org/en/latest/R-package/fiveMinutesNeuralNetwork.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dmlc/mxnet/pull/333/files#r42458228>.
>